src/config/config_type.rs

```diff
@@ -407,6 +409,14 @@ macro_rules! create_config {
                     "single_line_if_else_max_width",
                 );
                 self.single_line_if_else_max_width.2 = single_line_if_else_max_width;
+
+                let single_line_let_else_max_width = get_width_value(
+                    self.was_set().single_line_let_else_max_width(),
+                    self.single_line_let_else_max_width.2,
+                    heuristics.single_line_let_else_max_width,
+                    "single_line_let_else_max_width",
+                );
+                self.single_line_let_else_max_width.2 = single_line_let_else_max_width;
```

Comment:
These config values seems to be a bit repetitive. Would be nice to do as a followup to extract this to a macro/function.

Reply:
Agreed that it's something we should improve in a follow up PR

Reply:
very much agreed. this is my work from long ago and i remember having the same thought while writing it, but wasn't sufficiently motivated to do the macro work.

not as part of this pr, but i think either a FIXME comment or an issue would be helpful to mark some TBD work on this, otherwise it's likely to remain forgotten unless/until someone has to touch this part of the code again